content/docs/user-guide/managing-external-data.md

```diff
@@ -142,3 +126,25 @@ $ dvc run -d data.txt \
 
 Note that as long as there is a `hdfs://...` path for your data, DVC can handle
 it. So systems like Hadoop, Hive, and HBase are supported!
+
+### Local file system path
+
+The default cache location is `.dvc/cache`, so there is no need to move it for
```

Comment:
not sure I understand this change. let's do this as a separate PR with some scope and context please

Reply:
Rolled back.